src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataLatestIT.java

```diff
@@ -914,9 +914,13 @@ public void testHandlingSpecialChars() throws SQLException {
         "create or replace table \"TEST\\1\\_1\" (\"C%1\" integer,\"C\\1\\\\11\" integer)");
     statement.execute("INSERT INTO \"TEST\\1\\_1\" (\"C%1\",\"C\\1\\\\11\") VALUES (0,0)");
     // test getColumns with escaped special characters in schema and table name
-    statement.execute("create or replace schema \"SPECIAL%_\\SCHEMA\"");
+    String specialSchemaSuffix = SnowflakeUtil.randomAlphaNumeric(5);
+    String specialSchema = "\"SPECIAL%_\\SCHEMA" + specialSchemaSuffix + "\"";
```

Comment:
I see it again and again. Maybe it is worth having `createRandomSchema` method, or even better - `withRandomSchema`, which takes code and drops the code at the end?

Reply:
I have added dropping schema at the end, making it easier to use also in different tests could be done in a separate PR
here we want to have a specific schema with special chars
I know it's not perfect but it fixes the test and we can move forward and then we can think about future refactors for all the cleanup cases

Reply:
Yeah, the function can (and maybe even should) take a prefix parameter. I'm not sure about fixing all tests inproperly. Maybe you can add such a helper function here, and use it in next PRs? WDYT?

Reply:
Check now